cmd/kogito/command/remove/runtime_services.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remove
+
+import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/shared"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/spf13/cobra"
+)
+
+type removeRuntimeServiceFlags struct {
+	namespace string
+}
+
+type removableRuntimeService struct {
+	name string
+	list v1alpha1.KogitoServiceList
+}
+
+type removeRuntimeServiceCommand struct {
+	context.CommandContext
+	flags          removeRuntimeServiceFlags
+	command        *cobra.Command
+	runtimeService removableRuntimeService
+	Parent         *cobra.Command
+}
+
+var removableRuntimeServices = []removableRuntimeService{
+	{
+		name: "data-index",
+		list: &v1alpha1.KogitoDataIndexList{},
+	},
+	{
+		name: "mgmt-console",
+		list: &v1alpha1.KogitoMgmtConsoleList{},
+	},
+	{
+		name: "jobs-service",
+		list: &v1alpha1.KogitoJobsServiceList{},
+	},
+}
+
+func initRemoveRuntimeServiceCommands(ctx *context.CommandContext, parent *cobra.Command) []context.KogitoCommand {
+	var commands []context.KogitoCommand
+	for _, removable := range removableRuntimeServices {
+		cmd := &removeRuntimeServiceCommand{
+			CommandContext: *ctx,
```

Comment:
Should add here also the alias.

I can do `kogito install management-console` but cannot do `kogito remove management-console`